Dockerfile

```diff
@@ -59,8 +59,8 @@ VOLUME /var/lib/rancher/k3s
 
 FROM build AS build-k8s-codegen
 ARG KUBERNETES_VERSION
-RUN git clone -b ${KUBERNETES_VERSION} --depth=1 https://github.com/kubernetes/kubernetes.git ${GOPATH}/src/github.com/kubernetes/kubernetes
-WORKDIR ${GOPATH}/src/github.com/kubernetes/kubernetes
```

Comment:
No need to potentially trick go that it is building an old-school GOPATH project.